commons/src/main/java/com/expedia/www/haystack/pipes/commons/kafka/KafkaStreamStarter.java

```diff
@@ -32,31 +33,25 @@
 
 import java.util.Properties;
 
-import static com.expedia.www.haystack.pipes.commons.Configuration.HAYSTACK_KAFKA_CONFIG_PREFIX;
-import static com.expedia.www.haystack.pipes.commons.Configuration.HAYSTACK_PIPE_STREAMS;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
 public class KafkaStreamStarter {
-    // move this to configuration later
-    private final long MAX_CLOSE_TIMEOUT_SEC = 30;
-
+    static final String STARTING_MSG_WITH_TO_TOPIC =
+            "Attempting to start stream pointing at Kafka [%s] from topic [%s] to topic [%s]";
+    static final String STARTING_MSG_WITHOUT_TO_TOPIC =
+            "Attempting to start stream pointing at Kafka [%s] from topic [%s]";
+    static final String STARTED_MSG = "Now started Stream %s";
     @VisibleForTesting
     static Factory factory = new Factory();
     @VisibleForTesting
     static Logger logger = LoggerFactory.getLogger(KafkaStreamStarter.class);
     @VisibleForTesting
     static ConfigurationProvider CONFIGURATION_PROVIDER = new Configuration().createMergeConfigurationProvider();
-
-    static final String STARTING_MSG_WITH_TO_TOPIC =
-            "Attempting to start stream pointing at Kafka [%s] from topic [%s] to topic [%s]";
-    static final String STARTING_MSG_WITHOUT_TO_TOPIC =
-            "Attempting to start stream pointing at Kafka [%s] from topic [%s]";
-    static final String STARTED_MSG = "Now started Stream %s";
-
-    private final HealthController healthController;
-
     public final Class<? extends KafkaStreamBuilder> containingClass;
     public final String clientId;
+    // move this to configuration later
```

Comment:
Pardon my lack of context on this, when would it be "later" ? or what's the trigger point to move this to configuration?